### DISCUSSION TITLE
Attribution Reporting API OT and feature policies

### DISCUSSION THREAD

**Discussion by lbdvt** [privacysandbox/privacy-sandbox-dev-support#24] [Replacing cross-site tracking] [answered]
Hi,

The Attribution Reporting API mentions feature policies both on source and destination (https://github.com/WICG/conversion-measurement-api/blob/main/event_attribution_reporting_clicks.md#publisher-side-controls-for-attribution-source-declaration).

Are these feature policies active for the Origin Trial, i.e. access to AR API is restricted through these policies?

Thanks

**Comment by maudnals** [comment, marked as answer]
Hi @lbdvt,
Thanks for your question. In order to use Attribution Reporting in **cross-origin iframes** to declare a click (source-side) or trigger an attribution (destination side), you need to enable the feature via the `attribution-reporting` Permissions-Policy. (Permissions-Policy is the new name for Feature-Policy).
```html
<iframe src="..." allow="attribution-reporting"></iframe>
```
> Are these feature policies active for the Origin Trial, i.e. access to AR API is restricted through these policies?

This is in effect in the Origin Trial, but note that it isn't related to the Origin Trial per se; this is a generic mechanism to prevent arbitrary third-parties from registering sources without the publisher's knowledge (or triggering attributions).

Also note: using the permissions policy (`attribution-reporting`) is **not** needed in a top-level document or in same-origin iframes.

Does this answer your question?

**Reply by lbdvt** [reply]
Yes, thanks @maudnals !
